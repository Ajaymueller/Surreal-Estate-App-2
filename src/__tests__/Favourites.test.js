import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Favourites from '../Components/Favourites';
import { MemoryRouter } from 'react-router-dom';
import "@testing-library/jest-dom/extend-expect";

const props = {
    userID: "userID",
};

describe("Favourites", () => {
    it("component renders with correct props", () => {
        const { asFragment } = render (
        <MemoryRouter><Favourites {...props}/></MemoryRouter> )
        expect(asFragment).toMatchSnapshot();
    });
});