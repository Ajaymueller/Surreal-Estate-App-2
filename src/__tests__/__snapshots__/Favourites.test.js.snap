exports[`Favourites component renders with correct props 1`] = `[Function]`;
